src/main/java/teamroots/embers/util/EmberGenUtil.java

```diff
@@ -6,8 +6,8 @@
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 import net.minecraftforge.oredict.OreDictionary;
-import teamroots.embers.ConfigManager;
 import teamroots.embers.RegistryManager;
+import teamroots.embers.config.ConfigMaterial;
 
```

Comment:
empty line

Reply:
thats the default formatting everything not minecraft related gets an emlty line before it's imported. No idea how to change this in IntelliJ.